.github/workflows/distribution-checks.yml

```
@@ -58,6 +58,8 @@ jobs:
           distribution: 'adopt'
       - name: Set up Gradle
         uses: gradle/actions/setup-gradle@v4
+      - name: Run Unittests
+        run: ./gradlew :distribution:distribution:test
       - name: Generate target and test APK
         run: ./gradlew :distribution:sample:app:assembleDebug :distribution:sample:app:assembleAndroidTest
```

```suggestion
        run: ./gradlew :distribution:distribution:test :distribution:sample:app:assembleDebug :distribution:sample:app:assembleAndroidTest
```
Put them all on one line in order to run them in parallel

---

thanks! done